src/app/components/feed/FeedImportDialog.js

```diff
@@ -61,8 +62,9 @@ const FeedImportDialog = ({
   };
 
   const onError = (error) => {
-    const errorMessage = getErrorMessageForRelayModernProblem(error) || <GenericUnknownErrorMessage />;
-    setFlashMessage(errorMessage);
+    const errorMessage = getErrorMessage(error);
```

Comment:
It's been a long time since I have worked on these functions but I thought this was the legacy one, not sure why this needed to replace the "ForRelayModern" version

Reply:
In this case it was because I was interested only in the message string, not a full component.